src/test/java/app/coronawarn/verification/VerificationApplicationExternalTest.java

```diff
@@ -230,11 +289,13 @@ public void callGetRegistrationTokenByGuid() throws Exception {
     log.info("process callGetRegistrationTokenByGuid() ");
     appSessionrepository.deleteAll();
     RegistrationTokenRequest request = new RegistrationTokenRequest(TestUtils.TEST_GUI_HASH, RegistrationTokenKeyType.GUID);
-    mockMvc.perform(post(TestUtils.PREFIX_API_VERSION + TestUtils.REGISTRATION_TOKEN_URI)
+    String ret = mockMvc.perform(post(TestUtils.PREFIX_API_VERSION + TestUtils.REGISTRATION_TOKEN_URI)
       .contentType(MediaType.APPLICATION_JSON)
+      .header("cwa-fake" ,"0" )
       .content(TestUtils.getAsJsonFormat(request)))
-      .andExpect(status().isCreated())
-      .andExpect(jsonPath("$.registrationToken").exists());
+      .andReturn().getAsyncResult().toString();
+    assertTrue(ret.contains("201"));
```

Comment:
It's getting even more worse

I think the usage of ``` asyncDispatch() ``` would be a much cleaner solution